core/src/main/java/io/kestra/plugin/core/kv/Get.java

```diff
@@ -115,8 +115,8 @@ private Optional<KVValue> getValueWithInheritance(RunContext runContext, String
     @Getter
     public static class Output implements io.kestra.core.models.tasks.Output {
         @Schema(
-            title = "Value retrieve for the key.",
-            description = "This can be of any type and will keep the same as when it was set."
+            title = "Value retrieve for the key",
+            description = "This can be of any type and will stay the same as when it was set."
```

Comment:
```suggestion
            title = "Value retrieved for the key",
            description = "This can be of any type and will stay the same as when it was set."
```